manage_projects.rb

```diff
@@ -104,6 +105,11 @@ def update_attributes(project)
     else
       value = get_non_blank(attribute)
```

Comment:
When updating a project, there's nothing to inform the user what their choices are. It's probably worth adding something here or to the README telling the user that their options are `subscription` or `resource group`.